controllers/operands/kubevirt_test.go

```diff
@@ -633,25 +664,127 @@ Version: 1.2.3`)
 
 				mdevConf := foundResource.Spec.Configuration.MediatedDevicesConfiguration
 				Expect(mdevConf).ToNot(BeNil())
-				Expect(mdevConf.MediatedDevicesTypes).To(HaveLen(2))
-				Expect(mdevConf.MediatedDevicesTypes).To(ContainElements("nvidia-222", "nvidia-230"))
+				Expect(mdevConf.MediatedDeviceTypes).To(HaveLen(2))
+				Expect(mdevConf.MediatedDeviceTypes).To(ContainElements("nvidia-222", "nvidia-230"))
 				Expect(mdevConf.NodeMediatedDeviceTypes).To(HaveLen(2))
-				Expect(mdevConf.NodeMediatedDeviceTypes[0].MediatedDevicesTypes).To(ContainElements("nvidia-223"))
+				Expect(mdevConf.NodeMediatedDeviceTypes[0].MediatedDeviceTypes).To(ContainElements("nvidia-223"))
 				Expect(mdevConf.NodeMediatedDeviceTypes[0].NodeSelector).To(HaveKeyWithValue("testLabel1", "true"))
-				Expect(mdevConf.NodeMediatedDeviceTypes[1].MediatedDevicesTypes).To(ContainElements("nvidia-229"))
+				Expect(mdevConf.NodeMediatedDeviceTypes[1].MediatedDeviceTypes).To(ContainElements("nvidia-229"))
 				Expect(mdevConf.NodeMediatedDeviceTypes[1].NodeSelector).To(HaveKeyWithValue("testLabel2", "true"))
 
 			})
-			It("should update the permitted host devices configuration from the HC", func() {
+			It("should propagate the mediated devices configuration from the HC with node selectors with new APIs", func() {
```

Comment:
Not sure about "new API". This is very clear today. But one day (tomorrow?) it won't be new anymore...

Reply:
yes. it makes sense: let's call them mediatedDeviceTypes (explicitly) and deprecated (for better visibility).